website/cue/reference/urls.cue

```diff
@@ -602,6 +602,7 @@ urls: {
 	vector_repo:                                "\(github)/vectordotdev/vector"
 	vector_roles:                               "/docs/setup/deployment/roles"
 	vector_route_transform:                     "/docs/reference/configuration/transforms/route"
+	vector_exclusive_route_transform:           "/docs/reference/configuration/transforms/exclusive_route"
```

Comment:
Nit: I think you can leave this off until we actually refer to this URL anywhere.